app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/views/BusinessStartDate.scala.html

```diff
@@ -41,7 +41,8 @@
             captionClasses = Some("govuk-caption-l")
         ))
 
-        <p class="govuk-body">@messages("business.start-date.line-1")</p>
+        <p class="govuk-body govuk-hint" id="business-start-date.line-1">@messages("business.start-date.line-1")</p>
+        <p class="govuk-body govuk-hint" id="business-start-date.line-2">@messages("business.start-date.line-2")</p>
```

Comment:
```suggestion
        <p class="govuk-body" id="business-start-date.line-1">@messages("business.start-date.line-1")</p>
        <p class="govuk-body" id="business-start-date.line-2">@messages("business.start-date.line-2")</p>
```

This content is not hint text, you can ignore the prototype looking like it is hint text. This content is information for the user to read about what they are entering.